pandas/indexes/multi.py

```diff
@@ -2102,7 +2204,9 @@ def equals(self, other):
             return True
 
         if not isinstance(other, Index):
-            return False
+            if not isinstance(other, tuple):
+                return False
+            other = Index([other])
```

Comment:
this is to allow equivalence testing of a tuple and a MI which basically a list-of-tuples (though impl is different).